src/main/java/org/example/gamePlay/GamePlay.java

```diff
@@ -0,0 +1,192 @@
+package org.example.gamePlay;
+
+
+import org.example.model.Card;
+import org.example.model.House;
+import org.example.model.Player;
+
+import java.util.Scanner;
+
+public class GamePlay {
+
+    private final DeckService deckService = new DeckService();
+    private final Validators validators = new Validators(new Scanner(System.in));
+    private final Player player;
+    private final House house;
+    private final Scanner scanner = new Scanner(System.in);
+    private int wager = 0;
+    private boolean wouldYouPlay = true;
+    private int remainingCards = 0;
+
+    public GamePlay(House house, Player player) {
+        this.house = house;
+        this.player = player;
+    }
+
+    public void start() {
+        String deckId = startNewGame();
```

Comment:
why is starting new game returning a deck Id?

Reply:
The variable deckId is obtained after fetching a new deck. I need this variable to draw cards and to check how many cards are left in the deck.